styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -152,6 +153,8 @@
   private static final String STYX_STATE_MANAGER_TICK_INTERVAL = "styx.state-manager.tick-interval";
   private static final String STYX_SCHEDULER_THREADS = "styx.scheduler-threads";
   private static final String STYX_ENVIRONMENT = "styx.environment";
+  private static final String STYX_EXECUTION_ENV_VARS = "styx.execution-env-vars";
```

Comment:
```suggestion
  private static final String STYX_EXECUTION_ENV_VARS = "styx.XXX.execution-env-vars";
```
I think that it would be good to add some prefix, like docker or similar to signal that this is a config not related to flyte executions